src/main/java/org/honton/chas/license/maven/plugin/ScopeMatcher.java

```diff
@@ -26,6 +26,9 @@ public ScopeMatcher(String scopes) {
    * @return true, if the artifact matches
    */
   public boolean isMatch(String scope) {
+    if (scope == null) {
```

Comment:
if scope is not declared, should we check against all scopes?  That seems safest.

Reply:
You're right.
Returning true is the better option.